.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,199 @@
+name: Build and Publish to GHCR and MCR
+
+on: [workflow_dispatch]
```

Comment:
why manual? would you want to auto trigger off when a tag is cut?

Reply:
I think this is just because of the way we've historically managed releases. The process involves manually updating the changelog, and then running this action which creates the release/tag as well as building and publishing the images. There might be a more streamlined approach (@Tatsinnit - anything to add/suggest?).

Reply:
All sounds good, just as moving forward solution +1, at the moment, given its just me or Pete are the maintainer, we do manual build and publish for release cadence. We can post this merge think carefully about what pros and cons are for publishing trigger, tag cut vs this way. I am not sure where I have read pros and cons, was wondering if we can have this discussion post this PR. Sharing this link as fyi https://keepachangelog.com/en/1.0.0/ 

What o you think guys?

Reply:
I reopened #132 to discuss this. Let's keep the current manual trigger and consider other options later.